project/plugins/jira/__init__.py

```diff
@@ -27,19 +29,54 @@ def inviteUser(email,configMap,allPermissions, plugin_tag, name):
     headers = {'Accept':'application/json',
                'Content-Type': 'application/json'
                }
-    create=requests.post(url+'/rest/api/2/user', headers=headers,auth=(user, password), data=data)
-    data={'name': email.split('@', 1)[0]}
-    data = json.dumps(data)
-
-    groups = getCLIgroups(configMap, plugin_tag, allPermissions)
-    for group in groups:
-        add=requests.post(url+'/rest/api/2/group/user?groupname='+group, auth=(user, password),headers=headers, data=data )
-
-    log = 'Jira: ' + email.split('@', 1)[0] + ' added to ' + plugin_tag + '\n'
-    instruction = inviteMessage(configMap, plugin_tag)
-    return getJsonResponse("Jira Server",email, log, instruction)
+    #create a user
+    create = requests.post(url + '/rest/api/2/user', headers=headers, auth=(user, password), data=data)
+    if create.status_code > 201:
+        if create.status_code == 400:
+            log = plugin_tag + ' error: User already exists. Coudl not create user ' + username
+        elif create.status_code == 401:
+            log = plugin_tag + ' error: User ' + name + ' is not authenticated'
```

Comment:
I'm not sure this error message is correct - don't you get a 401 when the user *making* the call is not authorized? In this case your error message says the user you are trying to create is not authorized...